internal/server/templates/index.html

```diff
@@ -4,7 +4,39 @@
 	<meta charset="utf-8" />
 	<title>scrape</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>
 	<style>
+		body, html {
+			margin: 0;
+			padding: 0;
+			height: 100vh;
+		}
+		.page-container {
+			display: flex;
+			flex-direction: column;
+			gap: 0.6rem;
+			box-sizing: border-box;
+			height: 100%;
+			padding: 0.6rem;
+		}
+		.controls-container {
+			background-color: #fff;
+			box-sizing: border-box;
+		}
+		.response-container {
+			width: 100%;
+			min-width: 768px;
+			flex: 1;
+			overflow: auto;
+			border: 0.1rem inset #ccc;
+			padding-block: 0.2rem;
+			padding-inline: 0.4rem;
+			box-sizing: border-box;
+		}
```

Comment:
Question/suggestion: Will users need to copy result contents?

If so, we might add `user-select: all` or another method to make the text inside more easily selectable for copy/pasting. 

```suggestion
		.response-container {
			width: 100%;
			min-width: 768px;
			flex: 1;
			overflow: auto;
			border: 0.1rem inset #ccc;
			padding-block: 0.2rem;
			padding-inline: 0.4rem;
			box-sizing: border-box;
			user-select: all
		}
```

Reply:
There's no use case for that right now, it is a good idea tho. At some point (probably not in the near future due to time constraints) I hope to break this down into a different sort of display

Reply:
I tried this out and decided that it's better to let the user copy text at-will from the box (like if you want an image url). I did make the param explicit with `text`.